src/forms/cashflow.js

```diff
@@ -103,7 +103,37 @@ const CashFlowRow = function ({ generic, timeState, updateClientValue, children
     </CashFlowContainer>
   );
 
-};  // End CashFlowRow{} Component
+};  // End CashFlowInputsRow{} Component
+
+
+const CashFlowDisplayRow = function ({ generic, value, timeState, children }) {
+
+  var baseVal      = value || timeState[ generic ],
+      colClassName = `cashflow-column`,
+      weekly       = toMoneyStr(baseVal / (4 + 1 / 3)),
+      monthly      = toMoneyStr(baseVal),
+      yearly       = toMoneyStr(baseVal * 12);
+
+  /** @todo Make label a link to the input row */
+  return (
+    <div className = { `cashflow cashflow-display` }>
+      <div className = { colClassName + ` ` + generic + ` output-number` } >
+        { weekly }
+      </div>
+      <div className = { colClassName + ` ` + generic + ` output-number` } >
+        { monthly }
+      </div>
+      <div className = { colClassName + ` ` + generic + ` output-number` } >
+        { yearly }
+      </div>
+      <div className = { colClassName + ` cashflow-column-last-child` }>
```

Comment:
It's a little weird to be using template strings here but not interpolation. Not something that needs to be fixed though.